internal/manager/config.go

```diff
@@ -113,6 +115,8 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.KongAdminAPIConfig.CACert, "kong-admin-ca-cert", "", `PEM-encoded CA certificate to verify Kong's Admin SSL certificate.`)
 
 	flagSet.StringSliceVar(&c.KongAdminAPIConfig.Headers, "kong-admin-header", nil, `add a header (key:value) to every Admin API call, this flag can be used multiple times to specify multiple headers`)
+	flagSet.UintVar(&c.KongAdminInitializationRetries, "kong-admin-init-retries", 3, "Number of attempts that will be made initially on controller startup to connect to the Kong Admin API")
+	flagSet.DurationVar(&c.KongAdminInitializationRetryDelay, "kong-admin-init-retry-delay", time.Second*10, "The time delay between every attempt (on controller startup) to connect to the Kong Admin API")
```

Comment:
Imagine this scenario: we get a failure on start and retry, but then the API becomes available 1 second later and we wait 10 seconds despite this. 10 seconds can be a long time in some environments for the data-plane to start receiving updates and setting routes. I think we should make this a 1 second delay, and make the default number of retries high enough to account for about a minute of wait time (60) in the default case.